tests/queries/0_stateless/02456_test_attach_partition_from_table_with_more_granular_partition_expression_data_not_split.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS source;
+DROP TABLE IF EXISTS destination;
+
+CREATE TABLE source (timestamp DateTime) engine=MergeTree ORDER BY tuple() PARTITION BY toYYYYMMDD(timestamp);
+CREATE TABLE destination (timestamp DateTime) engine=MergeTree ORDER BY tuple() PARTITION BY toYYYYMM(timestamp);
+
+INSERT INTO TABLE source VALUES ('2010-03-02 02:01:01'), ('2010-03-02 02:01:03');
+
+ALTER TABLE destination ATTACH PARTITION '20100302' FROM source;
```

Comment:
>fasttest won't pass with this log statement, is that actually expected?

Yes, since now checksum will not be used and will be recalculated, you can omit this by putting `SET send_logs_level='error'` before alter, and restore it back to `warning` afterwards.